migration/template/apply/80-apply.sh

```diff
@@ -18,21 +18,35 @@ skipped_wc=(
 )
 
 run() {
+  local cluster="${2-both}"
```

Comment:
Use the environment variable instead of an argument if you go with an environment variable.